views/includes/documents/Chromium.md

```diff
@@ -12,7 +12,7 @@ To run userscripts on Chromium, you normally need a manager extension such as Ta
 
 [githubFavicon]: https://assets-cdn.github.com/favicon.ico
 [oujsFavicon]: https://raw.githubusercontent.com/OpenUserJs/OpenUserJS.org/master/public/images/favicon16.png
-[chromiumBrowser]: http://dev.chromium.org/Home
-[wikipediaChromium]: http://www.wikipedia.org/wiki/Chromium_%28web_browser%29
+[chromiumBrowser]: https://dev.chromium.org/Home
+[wikipediaChromium]: https://www.wikipedia.org/wiki/Chromium_%28web_browser%29
```

Comment:
This site has a preference of upgrading to secure. +1

This site has a preference of http with optional https. -1

Comment by @Martii
Originally posted at https://github.com/GreenLunar/OpenUserJS.org/commit/df9d5b50022cdbecfeb96fcf1daca1a07459ef16#commitcomment-16372961